src/main-client.ts

```diff
@@ -3348,15 +3399,40 @@ export class MainClient extends BaseRestClient {
     return this.getPrivate('sapi/v1/broker/transfer/futures', params);
   }
 
-  universalTransferBroker(
-    params: UniversalTransferBrokerParams,
-  ): Promise<BrokerSubAccount> {
+  getBrokerSubDepositHistory(
+    params: GetSubAccountDepositHistoryParams,
+  ): Promise<SubAccountDepositHistory[]> {
+    return this.getPrivate('sapi/v1/broker/subAccount/depositHist', params);
+  }
+
+  getBrokerSubAccountSpotAssets(
+    params: QuerySubAccountSpotMarginAssetInfoParams,
+  ): Promise<SubaccountBrokerSpotAssets> {
+    return this.getPrivate('sapi/v1/broker/subAccount/spotSummary', params);
+  }
+
+  getSubAccountMarginAssetInfo(
+    params: QuerySubAccountSpotMarginAssetInfoParams,
+  ): Promise<SubAccountBrokerMarginAssets> {
+    return this.getPrivate('sapi/v1/broker/subAccount/marginSummary', params);
+  }
+
+  querySubAccountFuturesAssetInfo(
+    params: QuerySubAccountFuturesAssetInfoParams,
+  ): Promise<BrokerFuturesSubAccountAssets> {
+    return this.getPrivate('sapi/v3/broker/subAccount/futuresSummary', params);
+  }
+
+  universalTransferBroker(params: UniversalTransferBrokerParams): Promise<{
```

Comment:
aha it is just below here, okay. Still dont handle git well haha

Reply:
yeah thats ok